benchmark/fluid/kube_gen_job.py

```diff
@@ -49,7 +49,10 @@ def parse_args():
     parser.add_argument(
         '--fluid', default=1, type=int, help='whether is fluid job')
     parser.add_argument(
-        '--rdma', action='store_ture', help='whether mount rdma libs')
+        '--rdma',
+        default=False,
+        action='store_true',
```

Comment:
`store_true` means default is false now, to the opposite `store_false` default is true.

Reply:
You are right!Done.